src/StateL1Null.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Null implementation for L1 state holder.
+ */
+
+namespace LCache;
+
+/**
+ * Fake/Stub state manager class.
+ *
+ * @author ndobromirov
+ */
+class StateL1Null extends StateL1Static implements StateL1Interface
```

Comment:
State NULL is deleted. On all places the static one is used now. The PHP_INT_MAX was moved back to the NullL1.